### ISSUE TITLE
nocaps dataset

### ISSUE THREAD

**Issue by yxoh** [salesforce/BLIP#50]
can you share the URL to download the NoCaps dataset or the code to download nocap images? The official website only provides the URL for each image.

**Comment by LiJunnan1992** [comment]
Hi, you can download from the URLs

**Comment by woctezuma** [comment]
See https://github.com/rom1504/img2dataset

**Comment by lorenmt** [comment]
Like to follow this issue again... 

I am wondering what's the best tool to convert urls to images?
I have downloaded the "ccs_synthetic_filtered_large.json" linked from this repo, while using requests.get() to download each url roughly resulted in 15% invalid urls across first 10k files I downloaded.

Is this normal? Or I should follow the package linked from @woctezuma , but if using that method, it would not easy to apply the filtered captions generated from BLIP right?

Really appreciate for the help and detailed instructions..
